sentry_sdk/integrations/modules.py

```diff
@@ -21,12 +21,20 @@
 def _generate_installed_modules():
     # type: () -> Iterator[Tuple[str, str]]
     try:
-        import pkg_resources
-    except ImportError:
-        return
+        from importlib.metadata import distributions, version
+
+        for dist in distributions():
+            yield dist.metadata["Name"], version(dist.metadata["Name"])
```

Comment:
There's also `dist.name` (it's just an alias of `dist.metadata["Name"]`), but it hasn't always been there. `dist.metadata["Name"]` should work with all py versions.